### PR TITLE
Create purikonejp

### PR DESCRIPTION
**Pull request by cky1201**
For Princess Connect (JP).

### CODE REVIEW

**Comment by IceCodeNew** [comment]
你看起来会讲中文，所以我就用中文回复了。
这个文件名我希望能再调整一下，同时你应该把它 include 进 `entertainment` 分类文件里。

**Comment by cky1201** [comment]
> 你看起来会讲中文，所以我就用中文回复了。
> 这个文件名我希望能再调整一下，同时你应该把它 include 进 `entertainment` 分类文件里。

Thank you for your reply. 
As a matter of fact, this is the first time a try to add file to a project. Please forgive me for any wrong doing. 
The name of this file would be shorten as to match its Japanese abbreviation. And I may re-create this file so as to merge into 'entertainment'.

**Comment by EpLiar** [comment]
cdn 的使用要加 full

File: data/purikonejp [hunk 1]
```diff
@@ -0,0 +1,3 @@
+api-priconne-redive.cygames.jp
+omotenashi.cygames.jp
+prd-priconne-redive.akamaized.net
```
Comment (IceCodeNew) [inline data/purikonejp:3]:
```suggestion
full:prd-priconne-redive.akamaized.net
```

**Comment by IceCodeNew** [comment]
Thanks! @cky1201

**Pull request merged**